packages/models/proto/v2/eservice/events.proto

```diff
@@ -33,6 +33,11 @@ message EServiceDraftDescriptorUpdatedV2 {
   EServiceV2 eservice = 2;
 }
 
+message EServiceDescriptorUpdatedV2 {
```

Comment:
as part of events refactor, we need to define also this event on a business use case.
Seeing the current use I could suggest `EServiceDescriptorQuotasUpdatedV2`, thoughts @beetlecrunch ?

Note: the draft update is generic because draft descriptor fields can be changed without business impacts

Reply:
@galales, that sounds good.
Let's rename the event.

Reply:
Renamed